src/tools/execution-results.h

```diff
@@ -100,17 +104,17 @@ struct ExecutionResults {
       auto name = iter.first;
       if (results.find(name) == results.end()) {
         std::cout << "[fuzz-exec] missing " << name << '\n';
-        abort();
+        return false;
```

Comment:
This function is `ExecutionResults::operator==(ExecutionResults& other)`, and I thought the natural thing an `operator==` would do when two things are not equal is to return `false` rather than to abort. And we have [code to abort if the results are not equal](https://github.com/WebAssembly/binaryen/blob/f2ba91b6340a8c3099ab6a48700c717cbd33599e/src/tools/execution-results.h#L92-L95) in the caller side too.